src/Reestr/Rep/ResponseInterface.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Wearesho\Bobra\Ubki\Reestr\Rep;
+
+use Wearesho\Bobra\Ubki\Reestr;
+
+/**
+ * Interface ResponseInterface
+ *
+ * @package Wearesho\Bobra\Ubki\Reestr\Rep
+ */
+interface ResponseInterface extends Reestr\ResponseInterface
+{
+    /**
+     * Reestr type
+     *
+     * @return string
+     */
+    public function getErtype(): string;
+
+    /**
+     * Criticality of error
+     * (CRITICAL - критичная, NOTICE - уведомление)
+     *
+     * @return string
+     */
+    public function getCrytical(): string;
+
+    /**
+     * Unique inn code of subject
+     *
+     * @return int
+     */
+    public function getInn(): int;
+
+    /**
+     * Note text
+     *
+     * @return string
+     */
+    public function getRemark(): string;
+
+    public function isErrorCritical(): bool;
```

Comment:
Is Error always exists?

Reply:
Attribute with error type always exist in response, but can be empty. I add function isNoError() for empty case in later commit